### PR TITLE
Privacy.md

### PR DESCRIPTION
**Pull request by HarryHeights**
This a PR for the Privacy policy. I'll begin work on the FAQs after some hours of rest. 
Please proof read and make necessary changes or otherwise point them out and I'll make the changes myself.

Thanks for your time.

### CODE REVIEW

**Comment by HarryHeights** [comment]
Okay what are the things to be changed so I make them right away.

**Comment by ScreamingHawk** [comment]
All those ^

I'm open to discussion if you have a different opinion. Thanks again for contributing this. I'm going to end up using it as a guide for other projects too. It's great!

**Comment by HarryHeights** [comment]
By all those you mean the whole privacy document?

**Comment by ScreamingHawk** [comment]
I've started a review in github. You can see it in this pull request above and in better detail if you use the `Files changed` tab of the pull request

**Comment by HarryHeights** [comment]
Yea I've seen it. But I really don't understand. You want to change some contents of the policy created? If that's what you mean it's no problem at all.

**Comment by ScreamingHawk** [comment]
Yes. Can you see each review note? They should alin directly with the line
in question.

**Comment by HarryHeights** [comment]
I can't see a review note

**Comment by ScreamingHawk** [comment]
Can you see these notes? 
image

**Comment by ScreamingHawk** [comment]
Oops! I hadn't clicked the `Submit Review` button. Hahaha. Try here: https://github.com/ScreamingHawk/phone-saver/pull/74/files/493fb53f34add0b7d4d40244d8795cf340a1b02d

**Comment by HarryHeights** [comment]
Okay. I can see where you want changes made now. I made some changes yesterday. I'll make further changes today.

File: Privacy.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+## Changes To This Privacy Policy
+
+This privacy policy statement is not final and is subject to changes at later times. This is done to accommodate changes in the software's architecture due to further developments and that these changes may not align with several statements and policies stated in the document. With that stated, you are advised to check this policy statement occasionally for any changes that may have occurred. Changes to this policy become effective as soon as they are made. <hr>
+
+
+## Disclaimer   
+
+Phone-Saver is developed in good faith to help end users copy and move and save any kind of file between storage locations swiftly and easily. The developer assumes no responsibility for any losses and/or damages of data that are in connection with the use of this software. 
+See [FAQs]() here if there are operations you don't understand. <hr>
```
[the remark this answers is not in the document]
Comment (HarryHeights) [reply Privacy.md]: Which line exactly?

**Comment by HarryHeights** [comment]
Hello. I've made changes according to your specifications. You can check them out again.

**Comment by ScreamingHawk** [comment]
This is great! Thanks so much for doing this. It's so good that I'll be using it as a template for my other project too

**Comment by HarryHeights** [comment]
You are welcome. I'm glad to be of help. I'm flattered also that you consider the work well done. If you need help with other projects too, I could also help out.

**Pull request merged**